src/manager.js

```diff
@@ -288,14 +288,14 @@ AdManager.prototype.onSlotRenderEnded = function (event) {
   if (event.isEmpty) {
     element.setAttribute('data-ad-load-state', 'empty');
   } else {
-
     if (this.adUnits.units[element.dataset.adUnit].onSlotRenderEnded) {
```

Comment:
It's actually the `if` block here that needs to get called regardless of `isEmpty`